crates/dump/src/lib.rs

```diff
@@ -271,13 +271,9 @@ impl<'a> Dump<'a> {
                     i = Indices::default();
                 }
 
-                Payload::CustomSection {
-                    name,
-                    data,
-                    data_offset,
-                } => {
+                Payload::CustomSection { name, data_offset, data, range } => {
                     write!(self.state, "custom section: {:?}", name)?;
-                    self.print(data_offset)?;
+                    self.print(range.start)?;
```

Comment:
I think `data_offset` here is technically correct given the current code, but that's only because the `write!` above accounts for all of the custom section id, the name, and the length of the whole section.

Could the `write!` above be split  and this calls `self.print` twice? The first `self.print` would simply be "custom section" with `range.start` as the initial start, and then the next `self.print` would be "name: {}" with the offset of `data_offset`